### ISSUE TITLE
Purse problems

### ISSUE THREAD

**Issue by snowboundmage** [CleverRaven/Cataclysm-DDA#44494]
Using the "sort armour" menu to unequip a purse with items in it causes game crash.
The purse was worn using the advanced inventory menu, and items were added in by unequipping other clothes from character (using sort armor menu) and automatically being put in the purse

- OS: Windows
    - OS Version: MINGW/CYGWIN/MSYS2 on Windows 7
- Game Version: 0.E-6346-g722966a [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food]
]

crash.log and debug.log
[config.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/5294961/config.zip)
save file(just the whole save folder, don't know what sepcifically i should zip)
[save.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/5294965/save.zip)

**Comment by wapcaplet** [comment]
Confirmed in 0.E-6518-g8e6077ef45, though it's a little tricky to make it occur. I think it has to do with having a `w`ielded item. If my hands are empty (nothing wielded), I can use `+` Sort armor to `u` remove a purse, and it goes in my hands. But if I am wielding a hammer, and try the same, the game crashes. I tried again, while wielding pliers, and the game did not crash - but instead got stuck in a seemingly infinite loop.
